game.lua

```diff
@@ -3052,10 +3052,17 @@ function drawentity(j, w, i, v, currentscissor, drop)
 			if v.graphiccolor and (not drop) then
 				love.graphics.setColor(v.graphiccolor)
 			end
+			if v.blending then
```

Comment:
We can probably infer that blending should be enabled if blendmode is defined (i.e., I think this should be `if v.blendmode`).